libs/cli/DOCS.md

```diff
@@ -12,18 +12,18 @@ $ langchain [OPTIONS] COMMAND [ARGS]...
 
 **Commands**:
 
-* `hub`: Manage installable hub packages.
-* `serve`: Manage LangServe application projects.
-* `start`: Start the LangServe instance, whether it's...
+* `app`: Manage LangServe application projects.
+* `serve`: Start the LangServe instance, whether it's...
```

Comment:
incomplete sentence

Reply:
This is just how typer autogens docs. Can debug later